### ISSUE TITLE
Cant have an empty progress bar

### ISSUE THREAD

**Issue by NathanHennigh** [kimmobrunfeldt/progressbar.js#213]
I want to use the progress bar to indicate how close a user is to advance to the next level. 
So far I tried to set .animate to zero, or remove it completely, but either way, it doesn't work.
It shows an empty bar but will not show the number within the circle.

**Comment by kimmobrunfeldt** [comment]
Could you please isolate the issue to a JSFiddle: http://jsfiddle.net/kimmobrunfeldt/8xa87k31/392/ ? To see what your existing code is and potentially there we can fix it. I can't guarantee when I'd have the next time to look into it though.

**Comment by YuFy1314** [comment]
> I want to use the progress bar to indicate how close a user is to advance to the next level.
> So far I tried to set .animate to zero, or remove it completely, but either way, it doesn't work.
> It shows an empty bar but will not show the number within the circle.

This problem also exists in my project. Is your problem solved?

**Comment by YuFy1314** [comment]
@NathanHennigh This problem also exists in my project. Is your problem solved?

**Comment by NathanHennigh** [comment]
No, I think I abandoned it all together after a while. It was a while ago

**Comment by YuFy1314** [comment]
It seems that I can only hide it by modifying the style of CSS.

**Comment by NathanHennigh** [comment]
Yeah, I honestly haven't looked at that code for a long time. But that's
sounds like one of the only valid solutions. Otherwise you could try and
edit the source file yourself or try and get in contact with whoever made it

**Comment by YuFy1314** [comment]
ok, thanks for the answer.

**Comment by over-engineer** [comment]
>It shows an empty bar but will not show the number within the circle.

For anyone reading this thread while trying to make [this example](https://jsfiddle.net/kimmobrunfeldt/72tkyn40/) always show the progress number (even when its value is 0), you have to remove the `if (value === 0)` check.

Change this:

```javascript
if (value === 0) {
  circle.setText('');
} else {
  circle.setText(value);
}
```

to that:

```javascript
circle.setText(value);
```
